index.js

```diff
@@ -2,38 +2,65 @@
 
 
 function calculate(steps) {
-  let total = 0
+  let valueOne
   let operator
+  let valueTwo
 
-  let valueTwo = 0
 
 
   for (let i = 0; i < steps.length; i++) {
     if (!isNaN(parseInt(steps[i])) && typeof (steps[i]) === 'string') {
-      total = parseInt(steps[i])
+      if (valueOne === undefined) {
+        valueOne = parseInt(steps[i])
+      } else {
+        valueTwo = parseInt(steps[i])
```

Comment:
If you used `Number()` instead of `parseInt()` then you wouldn't need special cases for `null` because `Number(null)` is `0`